continuous-delivery/test-pip-install.py

```diff
@@ -1,7 +1,9 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0.
 
-import awscrt.io
+import awscrt.s3
 
-print('if the next statement does not explode, the pip install was successful') 
-print('Is alpn supported? {}', awscrt.io.is_alpn_available())
+print("if the next statement does not explode, the pip install was successful")
+print(f"Is on Ec2 Instance nitro ? {awscrt.s3.is_running_on_ec2_nitro()}")
+print(f"Instance Type = {awscrt.s3.get_ec2_instance_type()}")
+print(f"Is Optimized for System = {awscrt.s3.is_optimized_for_system()}")
```

Comment:
revert this
it makes it seem like our release pipeline somehow how depends on checking that it's running on EC2